server/src/migration/1625762355386-ModifyObservation.ts

```diff
@@ -0,0 +1,18 @@
+import {MigrationInterface, QueryRunner} from 'typeorm';
```

Comment:
Apparently we don't have an eslint rule setup for this (I'll make a note to check that out), but the style we've been using is...

```typescript
import { SingleThing } from 'package';
import {
  FirstThing,
  SecondThing,
} from 'package';
```

Reply:
Will be fixed by eslint.